pvt/common/setupTests.ts

```diff
@@ -101,28 +101,28 @@ chai.use(function (chai, utils) {
           if (!matches || matches.length !== 6) throw error;
 
           const expectedReason: string = matches[2];
-          let actualErrorCode: string = matches[4];
-
-          // If the actual error code is not a balancer error, re-throw error
-          if (!actualErrorCode.includes('BAL#')) throw error;
-          actualErrorCode = actualErrorCode.replace('BAL#', '');
-
-          // @ts-ignore
-          let actualReason = reasons[actualErrorCode];
-          if (!actualReason) actualReason = 'Could not match a Balancer error message';
+          const actualErrorCode: string = matches[4];
+
+          let actualReason: string;
+          try {
+            actualReason = parseBalancerErrorCode(actualErrorCode);
+          } catch {
+            // If it's not a Balancer error then rethrow
+            if (!error.includes('BAL#')) return error;
```

Comment:
I'm not sure about this line - could you explain? Shouldn't it be before the `try`, and `throw` instead of `return`?

Reply:
Yeah I think this was relying on some throwing behaviour I had in `parseBalancerErrorCode` which I then stripped out. I've changed this up to make it more explicit. (`return` instead of `throw` is just a brainfart)